mycli/main.py

```diff
@@ -419,8 +419,9 @@ def handle_editor_command(self, cli, document):
         saved_callables = cli.application.pre_run_callables
         while special.editor_command(document.text):
             filename = special.get_filename(document.text)
-            sql, message = special.open_external_editor(filename,
-                                                          sql=document.text)
+            sql, message = special.open_external_editor(
+                filename, sql=document.text,
+                default_text=self.get_last_query() or '')
```

Comment:
Yeah, I'd like to streamline this a bit. I'll think through it.

`open_external_editor` is where the \e is removed, which `handle_editor_command` doesn't know about.